fastlane/lib/fastlane/actions/increment_version_number.rb

```diff
@@ -23,20 +23,24 @@ def self.run(params)
           '&&'
         ].join(' ')
 
-        current_version = `#{command_prefix} agvtool what-marketing-version -terse1`.split("\n").last || ''
+        begin
+          current_version = Actions
+                            .sh("#{command_prefix} agvtool what-marketing-version -terse1", log: FastlaneCore::Globals.verbose?)
+                            .split("\n")
+                            .last
```

Comment:
We _may_ also need to call `.strip` here with the addition of `^` and `$` to the regex. Thoughs? 🤔

Reply:
So far `agvtool` does not put any space around but it's definitely safer to put that! Will do.

Reply:
Done!